File: js/taller.js/28.js
let notaCualitativa = parseFloat(prompt("Ingrese la nota (0-5):"));

if (notaCualitativa >= 4.5 && notaCualitativa <= 5.0) {
    console.log("Excelente");
} else if (notaCualitativa >= 4.0 && notaCualitativa < 4.5) {
    console.log("Bueno");
} else if (notaCualitativa >= 3.0 && notaCualitativa < 4.0) {
    console.log("Aprobado");
} else if (notaCualitativa >= 0.0 && notaCualitativa < 3.0) {
    console.log("Reprobado");
} else {
    console.log("Nota fuera de rango válido.");
}

File: py/taller.py/28.py
nota = float(input("Ingrese la nota (0-5): "))

if 4.5 <= nota <= 5.0:
    print("Excelente")
elif 4.0 <= nota < 4.5:
    print("Bueno")
elif 3.0 <= nota < 4.0:
    print("Aprobado")
elif 0.0 <= nota < 3.0:
    print("Reprobado")
else:
    print("Nota fuera de rango válido.")